frappe/www/login.html

```diff
@@ -13,7 +13,7 @@
 	<div class="login-content page-card" style="margin-top: 30px;">
 		<form class="form-signin form-login" role="form">
 			<div class="page-card-head">
-				<span class="indicator blue" data-text="{{ _("Sign In") }}"></span>
+				<span class="indicator blue" data-text="{{ _("Login to start your session") }}"></span>
```

Comment:
seriously? why not just "Login"?

Reply:
@netchampfaris Did that to fix this https://github.com/frappe/frappe/pull/6664#issuecomment-449697358

can you suggest any other text?

Reply:
I don't think keeping the same text is a problem:

![image](https://user-images.githubusercontent.com/9355208/50555166-5b003200-0cee-11e9-8468-800ac800569b.png)

Reply:
If you really wanna change it, how about "Login to ERPNext" ?

Reply:
@netchampfaris I'll change back to "Login" then..